packages/console/src/mdx-components/Steps/FurtherReadings.tsx

```diff
@@ -15,11 +15,18 @@ function FurtherReadings(props: Props, ref?: Ref<HTMLDivElement>) {
     <Step ref={ref} {...stepProps}>
       <ul>
         {fullGuide && (
-          <li>
-            <TextLink href={fullGuide.url} targetBlank="noopener">
-              {fullGuide.title}
-            </TextLink>
-          </li>
+          <>
+            <li>
+              <TextLink href={`${fullGuide.url}/#get-user-information`} targetBlank="noopener">
```

Comment:
some guides does not have these two sections

Reply:
I'll take a look and have it displayed conditionally